Makefile

```diff
@@ -218,6 +220,10 @@ recover:
 .PHONY: ioctl
 ioctl:
 	$(GOBUILD) -ldflags "$(PackageFlags)" -o ./bin/$(BUILD_TARGET_IOCTL) -v ./tools/ioctl
+
+.PHONY: newioctl
```

Comment:
add cmd for cross building, like `ioctl-cross`

Reply:
ok